portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,16 +36,19 @@
 public class DataServlet extends HttpServlet {
   private final String COMMENT = "comment";
   private final String NAME = "name";
-  
+  private final String MAX_COMMENTS_LOAD = "loadcommentquantity";
```

Comment:
How about making this an "int" instead of a String. It is essentially an integer.